s2core/src/main/scala/com/kakao/s2graph/core/storage/hbase/AsynchbaseQueryBuilder.scala

```diff
@@ -220,7 +220,7 @@ class AsynchbaseQueryBuilder(storage: AsynchbaseStorage)(implicit ec: ExecutionC
           parentEdge <- prevStepEdgesOpt.get
         } yield parentEdge
 
-        fetch(queryRequest, prevStepScore, isInnerCall = true, parentEdges)
+        fetch(queryRequest, prevStepScore, isInnerCall = false, parentEdges)
```

Comment:
nice catch. do you think we should remove isInnerCall paramerter on fetch method also? just curious.